jmdaemon/test/test_daemon_protocol.py

```diff
@@ -59,11 +59,11 @@ def connectionMade(self):
 
     def clientStart(self):
         self.sigs_received = 0
-        irc = get_irc_mchannels()
+        irc = [get_mchannels()[0]]
```

Comment:
Nit in  5448a2631163c9bba5ca722a0628b33af717cdcd:

```suggestion
        chan_config = [get_mchannels()[0]]
```
Can we rename this?